Content.Client/_Starlight/ItemSwitch/ItemSystem.cs

```diff
@@ -0,0 +1,30 @@
+﻿using Content.Shared.Starlight.ItemSwitch;
+using Content.Shared.Interaction.Events;
+using Content.Shared.Interaction;
+using Content.Shared.Item;
+using Content.Shared.Item.ItemToggle.Components;
```

Comment:
```suggestion
using Content.Shared._Starlight.Item;
using Content.Shared._Starlight.Item.ItemToggle.Components;
```

Also rename this file to ItemSwitchSystem.cs.